### PR TITLE
AddPlaygroundAccess [finished, ready for merging]

### LINKED ISSUE #1114: Quest suggestion - Access to playgrounds
### General
Affected tag(s) to be modified/added: [](https://wiki.openstreetmap.org/wiki/Tag:leisure%3Dplayground)
Question asked: **Is this playground private or public??**

### Checklist
Checklist for quest suggestions (see guidelines):
- [x] 🚧 To be added tag is established and has a useful purpose
- [x] 🤔 Any answer the user can give must have an equivalent tagging (Quest should not reappear to other users when solved by one)
- [x] 🐿️ Easily answerable by everyone from the outside but a survey is necessary 
- [x] 💤 Not an overwhelming percentage of elements have the same answer (No spam)
- [x] 🕓 Applies to a reasonable number of elements (Worth the effort)

There are 455 486 objects (https://taginfo.openstreetmap.org/tags/leisure=playground) 2.96% have an access tag. At least in my area there are quite a lot of private playgrounds (mostly belonging to kindergartens) without proper access tag and when filtering out playgrounds in ie Osmand shows them as well.

Values yes, private or permissive (?) should be added to the access tag.

### PR DESCRIPTION
**Pull request by matkoniecz**
fixes #1114

### CODE REVIEW

File: app/src/main/java/de/westnordost/streetcomplete/quests/playground_access/AddPlaygroundAccess.java [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+	public void applyAnswerTo(Bundle answer, StringMapChangesBuilder changes)
+	{
+		if(answer.getBoolean(YesNoQuestAnswerFragment.ANSWER)){
+			changes.add("access", "yes");
+		} else {
+			changes.add("access", "private");
+		}
+	}
+	@Override public String getCommitMessage() { return "Add whatever playground is public"; }
```
Comment (ENT8R) [inline app/src/main/java/de/westnordost/streetcomplete/quests/playground_access/AddPlaygroundAccess.java]: Should not this be "Add **whether** playground is public"?

Comment (matkoniecz) [reply app/src/main/java/de/westnordost/streetcomplete/quests/playground_access/AddPlaygroundAccess.java]: I simplified the text here

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -609,5 +609,6 @@ Otherwise, you can download another keyboard in the app store. Popular keyboards
     <string name="quest_address_answer_no_housenumber_message2">Is this correct and is it not just part of a building?</string>
 
     <string name="notification_channel_download">"Download"</string>
+    <string name="quest_playground_access_title">Is this playground generally accessible?</string>
```
Comment (westnordost) [inline app/src/main/res/values/strings.xml]: *publicly

Comment (matkoniecz) [reply app/src/main/res/values/strings.xml]: done

**Comment by westnordost** [comment]
The drawables are there now, btw

**Comment by matkoniecz** [comment]
Thanks for the icon!

**Pull request merged**